feeds/README.md

```diff
@@ -0,0 +1,44 @@
+# Feeds
+
+Each of the feeds have their own implementation and support their own set of configuration options.
+
+## Configuration options
+
+`packages` this configuration option is only available on certain feeds, check the README of the feed you're interested in for information on this.
+
+`schedule` this allows for polling to be configured on a schedule for this specific feed. This is supported by all feeds. The value for this field should match the format defined by the [cron library](https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format), intervals and predefined schedules are also supported.
```

Comment:
Having the main option of  `poll_rate` parsed as time.Duration and feed specific schedules as cron formatted strings might be slightly confusing for users. Do we want to be consistent here? `poll_rate` has the caveat that it's used as an actual duration value too for the cutoff.

Reply:
Hmmm... perhaps it's worth restricting this value to be intervals set as time.Duration parsable strings which we can prepend with `@every`.

I think this is flexible enough for most usecases.

Reply:
I'd vote for the prepending where necessary :)

Reply:
renamed `schedule` to `poll_rate` and changed logic to operate identically to poll_rate in the top level configuration.